### ISSUE TITLE
CHS jsPsych init function

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#1282]
# TL;DR
- Provide jsPsych users with appropriate feedback when necessary trials aren't included in the experiment.
- Provide a way for a jsPsych experiment to automatically send responses back to our server/database (see also #1283).

# Narrative
To enforce (or simply show a warning) when a user doesn't include the needed trials in their experiment, we'll have to overwrite the jsPsych init function or the init's run function. This will allow us to check the trials array to make sure the appropriate CHS trials are present.  

A custom CHS version of the jsPsych init function will also allow us to automatically save the response data using the [jsPsych event-related callback functions](https://www.jspsych.org/7.3/reference/jspsych/#parameters), such as `on_data_update` and `on_finish` (see also #1283).

# Acceptance Criteria
- [ ] This overwriting of the init function needs to work with all jsPsych v7 versions
- [ ] Test across many different versions of v7 jsPsych
- [ ] Documentation

# Implementation Notes
Instead of a fork, It would be nice to have this be an additional package.  Maybe loaded after jsPsych. It'd be best for the user to not have to use a custom named function to initialize jsPsych.

It would also be nice if the custom jsPsych init function allowed researchers to use all of the parameters, including those that we are also using to handle the data. For instance, if we are automatically saving data via the `on_data_update` callback in jsPsych init, then ideally the user would also be able to use that same callback when they set up their experiment:
```js
var jsPsych = initJsPsych({
  on_data_update: function(data) {
    console.log('Just added new data. The contents of the data are: '+JSON.stringify(data));
  }
});
```
